ds_judgements_public_ui/templates/layouts/base.html

```diff
@@ -4,6 +4,8 @@
 <html lang="{{ LANGUAGE_CODE }}">
   <head>
     <meta charset="utf-8" />
+    {% block meta_description %}
```

Comment:
Do we want this on every single page?

Reply:
There's a good argument for us having a 'fallback' position here, but I don't feel incredibly strongly about it.